github/serverless/dagster-plus-deploy.yml

```diff
@@ -24,7 +24,7 @@ env:
 jobs:
   dagster_cloud_default_deploy:
     name: Dagster Serverless Deploy
-    runs-on: ubuntu-20.04
```

Comment:
probably also want to fix https://github.com/dagster-io/dagster-cloud-action-test/blob/main/templates/deploy.yml#L22